tools/run-tests.py

```diff
@@ -323,6 +324,12 @@ def run_buildoption_test(options):
 
     return ret
 
+def run_coverity_scan(options):
```

Comment:
That would be simplier, yes. I think about it, but I'm not so familiar with the travis config, and it's needed by the `script` command in travis.yml at line 71. Do you have any suggestion how to avoid it?